catalog/CSIC/CSIC_SPEI_2_9.jsonnet

```diff
@@ -431,8 +431,8 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
   'sci:citation': |||
```

Comment:
could you please add
'gee:skip_indexing': true
(example: https://github.com/google/earthengine-catalog/blob/main/catalog/GOOGLE/GOOGLE_GLOBAL_CCDC_V1.jsonnet#L15)

We need this to be set until the catalog images are generated.